apps/chat/src/store/models/models.epics.ts

```diff
@@ -153,7 +153,7 @@ const getInstalledModelIdsEpic: AppEpic = (action$, state$) =>
                 getRootId({ featureType: FeatureType.Application }),
               ),
             )
-            .map((app) => app.id);
+            .map((app) => app.reference);
           const installedModelIds = new Set(
             installedModels.map((model) => model.id),
```

Comment:
maybe lets try to use references here too? (157 line)

Reply:
there is no reference, it is installedModels array, which have { id: string; pinned: boolean; } type. It is already references.